fmriprep/workflows/bold/confounds.py

```diff
@@ -373,18 +374,23 @@ def init_ica_aroma_wf(template, metadata, mem_gb, omp_nthreads,
 
     The following steps are performed:
 
-    #. Smooth data using SUSAN
-    #. Run MELODIC outside of ICA-AROMA to generate the report
+    #. Smooth data using FSL `susan`, with a kernel width FWHM=6.0mm.
+    #. Run FSL `melodic` outside of ICA-AROMA to generate the report
     #. Run ICA-AROMA
     #. Aggregate identified motion components (aggressive) to TSV
-    #. Return classified_motion_ICs and melodic_mix for user to complete
-        non-aggressive denoising in T1w space
+    #. Return ``classified_motion_ICs`` and ``melodic_mix`` for user to complete
+       non-aggressive denoising in T1w space
     #. Calculate ICA-AROMA-identified noise components
-        (columns named ``AROMAAggrCompXX``)
+       (columns named ``AROMAAggrCompXX``)
 
     Additionally, non-aggressive denoising is performed on the BOLD series
     resampled into MNI space.
 
+    There is a current discussion on whether other confounds should be extracted
+    before or after denoising `here <http://nbviewer.jupyter.org/github/poldracklab/\
+    fmriprep-notebooks/blob/922e436429b879271fa13e76767a6e73443e74d9/issue-817_\
+    aroma_confounds.ipynb>`__.
```

Comment:
A couple relevant issues to this discussion: #1049, #1136.

I'm not sure it's a good idea to include these somehow, but just want to rope those discussions in here.

Reply:
Those issues are cited in the notebook @jdkent wrote (and pointed by the link). I assumed that was the best route to them as James introduce them with some context.

Reply:
Ah, cool. I didn't actually check the notebook. Lgtm.